packages/oas-utils/src/entities/spec/server.ts

```diff
@@ -37,7 +37,7 @@ export const oasServerSchema = z.object({
    * the host location is relative to the location where the OpenAPI document is being served. Variable substitutions
    * will be made when a variable is named in {brackets}.
    */
-  url: z.string().optional(),
+  url: z.string().optional().default(''),
```

Comment:
These types should be purely spec compliant. They can be defaulted when created.

Reply:
this fixes the type thats generated